include/zephyr/bluetooth/bluetooth.h

```diff
@@ -685,6 +685,28 @@ enum {
 	 * @note Requires @ref BT_LE_ADV_OPT_USE_NAME
 	 */
 	BT_LE_ADV_OPT_FORCE_NAME_IN_AD = BIT(18),
+
+	/**
+	 * @brief Use new random address on every connectable advertising
+	 * start.
+	 *
+	 * By default, when starting advertising using the @ref
+	 * BT_LE_ADV_OPT_CONNECTABLE option and @kconfig{CONFIG_BT_PRIVACY} is
+	 * disabled, the advertiser set uses the identity address as the
+	 * advertising address.
```

Comment:
If this is the case, then what is the purpose of `BT_LE_ADV_OPT_USE_IDENTITY`? 

You write `This option is the opposite to BT_LE_ADV_OPT_USE_IDENTITY` - If that is the case, then what is the difference between `BT_LE_ADV_OPT_USE_IDENTITY` not being set, and `BT_LE_ADV_OPT_USE_RANDOM` being set?

Reply:
Yes, this looks strange. But changing the current behavior will require users to update their advertiser options to get back the old behavior and this I wanted to avoid. Or do you think it should be done this way?

Reply:
Have we documented the behavior of not using `BT_LE_ADV_OPT_USE_IDENTITY` in a way that needs changing? 

I agree that we should be wary of changing behavior, but in the cases where the behavior is not documented, it's IMO less harmful.